sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -42,39 +34,19 @@ class ProjectOverviewView extends VerticalLayout{
         titleLabel.addStyleName(ValoTheme.LABEL_LARGE)
         projectGrid = new Grid<>()
 
-        detailsButton = new Button("Show Details",VaadinIcons.INFO_CIRCLE_O)
-        detailsButton.addStyleName(ValoTheme.LABEL_LARGE)
-        detailsButton.enabled = false
-        downloadButton = new Button("Download Batch Data",VaadinIcons.DOWNLOAD)
-        downloadButton.addStyleName(ValoTheme.LABEL_LARGE)
-        downloadButton.enabled = false
-
-        HorizontalLayout buttonLayout = new HorizontalLayout(detailsButton,downloadButton)
-
-        this.addComponents(titleLabel,buttonLayout,projectGrid)
+        this.addComponents(titleLabel, projectGrid)
     }
 
     private void fillGrid(){
-        projectGrid.addColumn({ it.projectCode })
+        projectGrid.addColumn({ it.projectId.projectCode.code})
```

Comment:
do we only want the project code or the whole ID including the space name?

Reply:
I asked a stakeholder and it seems not to be as important. I would wait until it will be requested by someone when seeing the release